### DISCUSSION TITLE
How to show a blank page when opening FreeTube?

### DISCUSSION THREAD

**Discussion by josipDogan** [FreeTubeApp/FreeTube#2761] [Q&A]
I don't want all my subscriptions to pop up when I opejn FreeTube. How to surpass this?
Thanks

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
So would u like an empty subscriptions tab or would u like to not like to see new videos when u start FT?

**Reply by josipDogan** [reply]
![image](https://user-images.githubusercontent.com/61889257/198146371-ac279b2c-17a3-4ebd-8a6c-cb4ea4ebbd97.png)

Like in the example in NewPipe. Totally empty page. Is it possible. Thanks!

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
The closest thing to this that is implemented in the latest nighty builds is that u can disable automatic subscription fetching on startup. At this time we don't think it is worth implementing a fully empty page.

**Reply by josipDogan** [reply]
Thanks!
